integration/RUNLOCAL

```diff
@@ -4,4 +4,10 @@
 # Some examples:
 # ./RUNLOCAL
 # ./RUNLOCAL -v  -test.run TestTczclient
```

Comment:
Will `-test.run TestTczclient` still work with this script? If not, that's fine, just remove this example.

Reply:
Should still work